sdcm/utils/git.py

```diff
@@ -23,3 +23,16 @@ def get_git_current_branch() -> str:
         LOGGER.warning("Error running git command", exc_info=True)
         return "#ERROR"
     return proc.stdout.decode(encoding="utf-8").strip()
+
+
+def clone_repo(remoter, repo_url: str):
```

Comment:
Would be nice by adding such a util, to also refactor another place that does the same and can earn from this.
(In a separate commit).
This is one of the ways to make sure other people are using the common utils.

Reply:
I refactored 2 places in which `git clone` was used directly (`jepsen_test.py`, `cluster.py`) in the final commit to this PR